src/helpers.ts

```diff
@@ -53,6 +53,9 @@ function getDay(timestamp: BigInt): BigInt {
 function getDayId(timestamp: BigInt): Bytes {
   return Bytes.fromI32(getDay(timestamp).toI32());
 }
+function getMarketDayId(marketAddress: Bytes, timestamp: BigInt): Bytes {
+  return Bytes.fromUTF8(`${marketAddress.toHexString()}-${getDay(timestamp).toString()}`);
```

Comment:
It's better to concat Bytes instead of using utf8 encoding. 
With concat, the ID is still readable 

```suggestion
  return marketAddress.concat(Bytes.fromI32(getDay(timestamp)));
```

Reply:
I wanted to have an ID like `address-day` instead of `addressday`, which I find less readable and parseable… Is there a way?